dmscripts/export_framework_results_reasons.py

```diff
@@ -194,6 +161,14 @@ def create_row(self, record):
             contact_details(record)
 
 
+def get_questions_numbers_from_framework(framework_slug, content_loader):
```

Comment:
Oh fanny I know what this is - it's usually due to trying to use the non-mutating update `dict(basedict, **updatedict)` trick when it turns out that one of `updatedict`'s keys isn't a string. Not entirely sure where in this case but I'll have to find it.